pyomo/solvers/tests/models/LP_unique_duals.py

```diff
@@ -8,7 +8,8 @@
 #  This software is distributed under the 3-clause BSD License.
 #  ___________________________________________________________________________
 
-from pyomo.core import ConcreteModel, Param, Var, Expression, Objective, Constraint, RangeSet, ConstraintList, NonNegativeReals, Suffix, summation
+from pyomo.core import ConcreteModel, Param, Var, Expression, Objective, Constraint, RangeSet, ConstraintList, \
+    NonNegativeReals, Suffix, summation, sum_product
```

Comment:
Same case here, do not edit any of the files that do not belong to the original contribution.
Were these changes required for the tests of communities to work?